### ISSUE TITLE
Support for GWT 2.9.0

### ISSUE THREAD

**Issue by salmonb** [foal/gwt-time#3]
Compiling gwt-time with elemental2 version 1.0.0 produces this error:

```
[INFO]    Tracing compile failure path for type 'org.gwtproject.core.client.ScriptInjector'
[INFO]       [ERROR] Errors in 'jar:file:/home/bruno/.m2/repository/org/jresearch/gwtproject/core/gwt-core/1.1/gwt-core-1.1.jar!/org/gwtproject/core/client/ScriptInjector.java'
[INFO]          [ERROR] Line 43: The method getAny(String) is undefined for the type JsPropertyMap<Any>
[INFO]          [ERROR] Line 108: The method getAny(String) is undefined for the type JsPropertyMap<Any>
[INFO]          [ERROR] Line 239: Void methods cannot return a value
[INFO]    Tracing compile failure path for type 'org.gwtproject.core.client.JsArrayMixed'
[INFO]       [ERROR] Errors in 'jar:file:/home/bruno/.m2/repository/org/jresearch/gwtproject/core/gwt-core/1.1/gwt-core-1.1.jar!/org/gwtproject/core/client/JsArrayMixed.java'
[INFO]          [ERROR] Line 40: The method getAnyAt(int) is undefined for the type JsArray<Object>
[INFO]          [ERROR] Line 63: The method getAnyAt(int) is undefined for the type JsArray<Object>
[INFO]          [ERROR] Line 51: The method getAnyAt(int) is undefined for the type JsArray<Object>
[INFO]    [ERROR] Aborting compile due to errors in some input files
```
I think this is because gwt-time has been designed with elemental2 version 1.0.0-RC1 and now elemental2 version 1.0.0 has some breaking changes that causes these errors.

Is it possible to upgrade gwt-time to elemental2 version 1.0.0?

**Comment by FrankHossfeld** [comment]
This problem is related to `org.jresearch.gwtproject.core:gwt-core:1.1` which has not been updated to work with Elemental2 1.0.0.

Once updated it will only work with Elemental2 1.0.0 and requires that all other third party libs will work with Elemental2 1.0.0

**Comment by salmonb** [comment]
Ok. Do you know where this project is hosted and if they work on this?

**Comment by salmonb** [comment]
This version of Elemental2 seems closer in my tests to 1.0.0 than 1.0.0-RC1 and it works with GWT 2.8.2: `org.realityforge.com.google.elemental2:elemental2-core:2.25`

Just in case it can help.

**Comment by foal** [comment]
Fixed in v 2.0.0
